paris-processor/src/main/java/com/airbnb/paris/processor/framework/KotlinPoetConverters.kt

```diff
@@ -184,6 +187,8 @@ internal fun Modifier.toKModifier() = when (this) {
     else -> null
 }
 
+internal fun JavaCodeBlock.toKPoet(): KotlinCodeBlock = KotlinCodeBlock.of(toString())
```

Comment:
does this always work correct with arguments?

Reply:
Thanks for pointing this out. I think the arguments part is probably fine but assuming that Java code can be used as is as Kotlin code isn't. I'll remove this and find a different solution.